docs/sources/http_api/annotations.md

```diff
@@ -251,3 +251,37 @@ Content-Type: application/json
     "message":"Annotation deleted"
 }
 ```
+
+## Find Annotations Tags
+
+`GET /api/annotations/tags`
+
+Find all the event tags created in the annotations
+
+**Example Request**:
+
+
+```http
+GET /api/annotations/tags HTTP/1.1
+Accept: application/json
+Content-Type: application/json
+Authorization: Basic YWRtaW46YWRtaW4=
+```
+
+**Example Response**:
+
+```http
+HTTP/1.1 200
+Content-Type: application/json
+
+{
+    "result": {
+        "tags": [
+            {
+                "tag": "outage",
+                "count": 1
+            }
+        ]
+    }
+}
+```
```

Comment:
A bit unsure if the tag-query is optional or not. Please verify.

```suggestion
Find all the event tags created in the annotations

**Example Request**:

```http
GET /api/annotations/tags?tag=out HTTP/1.1
Accept: application/json
Content-Type: application/json
Authorization: Basic YWRtaW46YWRtaW4=
```

Query Parameters:
- `tag`: string. Optional. Used to filter tags.
- `limit`: number. Optional - default is 100. Max limit for results returned.

**Example Response**:

```http
HTTP/1.1 200
Content-Type: application/json

{
    "result": {
        "tags": [
            {
                "tag": "outage",
                "count": 1
            }
        ]
    }
}
```
```

Reply:
@marefr Thank you for taking a look 🙌🏾, indeed the tag query is optional (I tested it), I will apply the suggestions